### ISSUE TITLE
Erlang 23.0 freezes

### ISSUE THREAD

**Issue by eproxus** [ish-app/ish#1385]
Installing the package `erlang-23.0-r0` hangs the prompt. App can be used but ^C does not work and the prompt can’t be recovered without killing the app.

**Comment by Mnpn03** [comment]
Hi! It seems like you mean that running `erl` hangs, not the installation itself, correct? If so, I can reproduce this, and it looks similar to #1121.

**Comment by eproxus** [comment]
@Mnpn03 Correct. Yeah, they look similar (both start the Erlang VM so it's most likely the same issue).

**Comment by foldl** [comment]
I can reproduce this too. I had tried Erlang 24 and 23.

Hope to see Erlang running withing `ish`.

**Comment by foldl** [comment]
Update:

With v1.3.2:

Erlang 23 can be started (failure occurs - segmentation fault - occussionaly). But `make:all()` will cause the shell hang definately. Maybe it is related to file I/O.
